docs/AWS-CREDENTIALS.md

```diff
@@ -3,5 +3,13 @@
 You'll need AWS credentials, which have [full JPT permissions].
 The order of loading credentials is defined in [`AwsVus`] in the `prepareAws` method.
 
+## AWS credentials for Atlassian engineering
+If you are a member of an Atlassian engineering departament/team,
+you can obtain AWS credentials via our internal tool - [cloudtoken].
+You can run it in daemon mode: `cloudtoken -d`
+and assume the `arn:aws:iam::695067801333:role/Jira-Server-Perf-Dev-Protection-Pipeline` role.
+It already has the permissions set up.
+
 [full JPT permissions]: https://bitbucket.org/atlassian/jira-performance-tests/src/master/aws-policy.json
 [`AwsVus`]: ../src/test/kotlin/jces1209/AwsVus.kt
+[cloudtoken]: http://go.atlassian.com/cloudtoken
```

Comment:
maybe let's add there `go/aws` as well?

Reply:
What's the use case for `go/aws`?

Reply:
I thought about quickly checking roles that you can assume, like before setting up `cloudtoken`

Reply:
We need `cloudtoken` anyway, and it will list the roles. I want to avoid inflating the process.